src/art/local/backend.py

```diff
@@ -379,3 +384,43 @@ async def _experimental_push_to_s3(
             delete=delete,
             art_path=self._path,
         )
+
+    async def _experimental_deploy(
```

Comment:
Let's add a required parameter `deploy_to`, which must be set to "together". Will help people understand what's going on.

Reply:
Added this class:

```
class LoRADeploymentProvider(str, Enum):
    TOGETHER = "together"
```